Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.xaml

```diff
@@ -0,0 +1,244 @@
+                                    <Button x:Name="EraseAllButton"
+                                            Style="{StaticResource CanvasTextBoxButtonStyle}"
+                                            ToolTipService.ToolTip="Erase All">
+                                        <Button.Content>
+                                            <SymbolIcon Symbol="Delete" />
+                                        </Button.Content>
+                                    </Button>
+                                    <Button x:Name="UndoButton"
+                                            Style="{StaticResource CanvasTextBoxButtonStyle}"
+                                            ToolTipService.ToolTip="Undo">
+                                        <Button.Content>
+                                            <SymbolIcon Symbol="Undo" />
+                                        </Button.Content>
+                                    </Button>
+                                    <Button x:Name="RedoButton"
+                                            Style="{StaticResource CanvasTextBoxButtonStyle}"
+                                            ToolTipService.ToolTip="Redo">
+                                        <Button.Content>
+                                            <SymbolIcon Symbol="Redo" />
+                                        </Button.Content>
+                                    </Button>
+                                </StackPanel>
+                                <StackPanel x:Name="CanvasTextBoxTools"
+                                            Background="{ThemeResource SystemChromeMediumColor}"
+                                            Orientation="Horizontal"
+                                            Visibility="Collapsed">
+                                    <Grid Width="2"
+                                          Height="48"
+                                          Background="Black" />
+                                    <Button Padding="0,2,0,0"
+                                            Style="{StaticResource CanvasTextBoxButtonStyle}"
+                                            ToolTipService.ToolTip="Text Color">
+                                        <Button.Content>
+                                            <FontIcon x:Name="FontColorIcon"
+                                                      FontFamily="Segoe MDL2 Assets"
+                                                      Foreground="Red"
+                                                      Glyph="&#xE8D3;" />
+                                        </Button.Content>
+                                        <Button.Flyout>
+                                            <Flyout Placement="Bottom">
+                                                <ColorPicker x:Name="CanvasTextBoxColorPicker"
+                                                             Color="Red" />
```

Comment:
Thought this was just for the sample app, not sure if we want Red text to be the default for all text on a default InifiniteCanvas.

Probably just best to set it back to black.  This is more an issue I have with the ColorPicker control itself.  I may file some feedback hub issues on it.

Reply:
Yeah but the black color picker is really misleading. I think i will create a property so the user can change the default color :) 
lets leave it at developer's side 😈